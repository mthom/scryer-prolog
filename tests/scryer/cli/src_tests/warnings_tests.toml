args = [
    "-f",
    "--no-add-history",
    "src/tests/warnings.pl",
    "src/tests/warnings1.pl",
    "src/tests/warnings2.pl",
    "src/tests/warnings3.pl",
    "-g", "halt"
]
