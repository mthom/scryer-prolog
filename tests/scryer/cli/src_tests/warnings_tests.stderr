% Warning: integer/1 is a constant source of wrong results, use integer_si/1 from library(si) at line 8 of warnings.pl
% Warning: Nested negations can be reduced at line 12 of warnings.pl
% Warning: Meta-predicate detected, but no qualified variables found at line 15 of warnings.pl
% Warning: Meta-predicate detected, but no qualified variables found at line 16 of warnings.pl
% Warning: Meta-predicate detected, but no qualified variables found at line 17 of warnings.pl
% Warning: (b,c) attempts to re-define (,)/2 at line 6 of warnings1.pl
% Warning: (/+c) attempts to re-define (/+)/1 at line 6 of warnings2.pl
% Warning: Arithmetic expression A is B+sqrt(-C*D+E) contains invalid terms [[1]=B,phi=C,max(3+5)=E] at line 4 of warnings3.pl
